.github/workflows/ota-update.yml

```diff
@@ -0,0 +1,29 @@
+on:
+  push:
+    tags:
+      # matches "v1.2.1.ota-prod.8"
+      - 'v[0-9]+.[0-9]+.[0-9]+ota-prod.[0-9]+'
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🏗 Setup repo
+        uses: actions/checkout@v3
+
+      - name: 🏗 Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: yarn
+
+      - name: 🏗 Setup EAS
+        uses: expo/expo-github-action@v8.2.1
+        with:
+          eas-version: latest
+          token: ${{ secrets.EXPO_UPDATE_TOKEN }}
+
+      - name: 📦 Install dependencies
+        run: yarn install
+
+      - name: 🚀 Create update
+        run: eas update --auto --branch production --non-interactive
```

Comment:
Why there is here a `--branch` and in [the file above](https://github.com/bratislava/enforcement-scan-app-native/pull/118/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721R29) is `--profile`? Shouldn't it be the same?

Reply:
the eas update and eas build work kind of differently and we have the wrong branch name in eas.json (it should be master) but that is another thing both on mpa and here (not easy to change, because the users are already there and if we change it some wont get it)

Reply:
Ok, thanks for the explanation